### PR TITLE
fix syslog port; add support for monitor agent, debug agent

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by lukas-vlcek** [comment]
LGTM

**Comment by lukas-vlcek** [comment]
I suggest to merge this and then we can improve the travis part similar to what I propose in https://github.com/ViaQ/docker-fluentd/pull/23
@richm let me know what you think, I can work on this today and merge it and prepare another PR.

File: configs.d/input/debug.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<source>
+  @type debug_agent
```
Comment (lukas-vlcek) [inline configs.d/input/debug.conf:2]: BTW, @richm did you find any information about how this `debug_agent` can be used? I do not think this is exposed via HTTP/REST in fluentd.

Comment (richm) [inline configs.d/input/debug.conf:2]: You have to expose port 24230 `docker ... -p 24230:24230`, then use the program `fluent-debug` provided with the fluentd distribution.  It's primarily for plugin developers, or for debugging really hard problems using `druby`.

**Pull request merged**